scd30/scd30_example_usage.c

```diff
@@ -58,15 +58,37 @@ int main(void) {
     scd30_set_measurement_interval(interval_in_seconds);
     sensirion_sleep_usec(20000u);
     scd30_start_periodic_measurement(0);
-    sensirion_sleep_usec(interval_in_seconds * 1000000u);
 
     while (1) {
+        uint16_t data_ready = 0;
+        uint16_t timeout = 0;
+
+        /* Poll data_ready flag until data is available. Allow %20 more than
```

Comment:
```suggestion
        /* Poll data_ready flag until data is available. Allow 20% more than
```
or was that intentional?